src/Dotnet.Script/Extensions.cs

```diff
@@ -0,0 +1,18 @@
+using System.IO;
+
+namespace Dotnet.Script
+{
+    using System;
```

Comment:
can we move the usings outside to be consistent?

Reply:
Done in 3316cc217ea8580c1014bb360f1c0ea776bcc94d.